### PR TITLE
High Resoution Using Retina Display

### LINKED ISSUE #7627: High Resolution Display using a Retina Image Concept
Different devices has different size of display for making high resolution we use retina image concept.

### PR DESCRIPTION
**Pull request by satyam-mahajan**
<hr>

## Description 📜

Apple made a retina display for high resolution which is more pixelated and much cleared than normal one. Let's we go to study in detail. 
Fixes #7627 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Ot#her (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# High Resolution Display using the Retina Imgae Concept
+
+The Apple has introduced a display called the "Retina Display" which is much better than our normal display such as picture element Per Inch(PPI) or Dots Per Inch(DPI).
+This display has high resolution power and we can easily distinguish between normal and retina display because each display vary sizes and specifications.
+
+### Display Resolution
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md:6]: Wrong level of heading, use ##.

File: Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The display resolution of the device such as laptop monitor,smart tv is that the variety of different pixels in every space that may be displayed.In digital mensuration, the display resolution run in pixels per inch (PPI).
+
+There are two types of display resolution we can define :-
+
+1) Low Display Resolution : The display in which Pixels per inch has low value is called low display resolution .It is Less sharper as compare to high resolution.
+2) High Display Resolution : The display in which Pixels per inch has higher value as compared to other is called the high display resolution and it is more clear.
+
+### Retina Display
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md:15]: Wrong level of heading, use ##.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Very less content, add a bit more.

**Comment by satyam-mahajan** [comment]
I do the changes ,please it again.

**Comment by kashishahuja2002** [comment]
> I do the changes ,please review it again @kashishahuja2002

Did you add a little more content, looks the same to me.

**Comment by satyam-mahajan** [comment]
No , I add one paragraph after you say.. Need I write some more? @kashishahuja2002

**Comment by satyam-mahajan** [comment]
I add some more content . Please can you review it @kashishahuja2002

**Review by kashishahuja2002 — APPROVED** [review]
LGTM. Tag the supervisor of your batch fir further reviewing and merging of the PR.

**Comment by satyam-mahajan** [comment]
please review my PR @Knighthawk-Leo

**Comment by Knighthawk-Leo** [comment]
Ok looking for it

**Comment by satyam-mahajan** [comment]
please review and merge my PR  @code-reaper08 .

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Hello @satyam-mahajan 👋 
There are many typos and grammatical mistakes, please fine tune them. I would also suggest you to add image source links right below the images.

**Comment by satyam-mahajan** [comment]
I do the required changes , please review it @code-reaper08 .

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Please format the spacings and still there are grammatical errors. Take your time and do the changes.

**Comment by satyam-mahajan** [comment]
I remove all grammatical mistakes ,please you review it once again @code-reaper08 .

**Comment by satyam-mahajan** [comment]
please review my PR @code-reaper08 .

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
@satyam-mahajan still there are more grammatical errors, I suggest you to use a service like **Grammarly** to clean up the errors.

**Comment by satyam-mahajan** [comment]
I remove all grammatical mistakes , please review it @code-reaper08

File: Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+A Retina display is a term created by Apple to describe devices and monitors that have a resolution and element density so high that someone is unable to recognise the individual pixels at a traditional viewing distance. Now, The Apple device's display was well received and greatly improved the viewing experience.
+
+Difference between Non Retina Display and Retina Display
+
+1. Retina Display Image
+
+![300px-Retina_Display](https://user-images.githubusercontent.com/54473091/142192529-16602e16-c339-44de-915d-ddc02687b69d.jpg)
+
+[Retina Display Image](https://user-images.githubusercontent.com/54473091/142192529-16602e16-c339-44de-915d-ddc02687b69d.jpg)
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md:24]: Place a link to the actual source.
AFAIK you've taken the images from [here](https://en.wikipedia.org/wiki/Retina_display).

Do update the captions to real links to source.

File: Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+2. Non Retina Display Image
+
+![300px-Non-Retina_Display](https://user-images.githubusercontent.com/54473091/142193055-64f38832-168c-45ed-94ee-d02bfb779dbb.jpg)
+
+[Non Retina Display Image](https://user-images.githubusercontent.com/54473091/142193055-64f38832-168c-45ed-94ee-d02bfb779dbb.jpg)
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md:30]: Same here,
Place a link to the actual source.
AFAIK you've taken the images from [here](https://en.wikipedia.org/wiki/Retina_display).

Do update the captions to real links to source.

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Kindly look through the review and change the captions.

Grammatical errors are now fixed 👍

**Comment by satyam-mahajan** [comment]
please review it again @code-reaper08 .

**Comment by satyam-mahajan** [comment]
Please review it @code-reaper08 . I do all the required changes.

**Comment by satyam-mahajan** [comment]
please review my PR @code-reaper08 and @Knighthawk-Leo

**Review by code-reaper08 — APPROVED** [review]
LGTM ✨

**Pull request merged**